apps/status/app/api/status/events/README.md

```
@@ -32,6 +32,14 @@ POST /api/status/events
 
 `services` または `incidents` の少なくとも一方を指定する必要があります。
 
+#### 再同期（リフレッシュ）
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)